prow/cmd/deck/jobs.go

```diff
@@ -207,10 +207,6 @@ func (ja *JobAgent) update() error {
 		buildID := j.Status.BuildID
 		nj := Job{
```

Comment:
I thought we were just going to get rid of this type?

Reply:
We will, there is an issue about it: https://github.com/kubernetes/test-infra/issues/5216